src/aleph/sdk/chains/common.py

```diff
@@ -120,7 +120,12 @@ def get_fallback_private_key(path: Optional[Path] = None) -> bytes:
         path.write_bytes(private_key)
 
         default_key_path = path.parent / "default.key"
+
+        # We check if the symlink exist but dead to remove it
+        if default_key_path.is_symlink() and not default_key_path.resolve().exists():
+            default_key_path.unlink()
+
+        # We create the symlink if symlink not exist
```

Comment:
Keep the old comment?
```suggestion
        # Create a symlink to use this key by default
```